client/post-editor/editor-drawer/index.jsx

```diff
@@ -261,6 +262,17 @@ const EditorDrawer = React.createClass( {
 		);
 	},
 
+	renderCopyPost: function() {
+		// TODO: replace with currentPostTypeSupports( 'copy' );
+		if ( 'post' !== this.props.type ) {
+			return;
+		}
+
+		return (
```

Comment:
Is fine if you think it helps readability, but just FYI that the parentheses aren't necessary (especially applicable to one-liner elements):

```jsx
return <EditorMoreOptionsCopyPost />;
```